include/bbp/sonata/report_reader.h

```diff
@@ -62,6 +67,8 @@ class SONATA_API SpikeReader
         Population(const std::string& filename, const std::string& populationName);
 
         Spikes spikes_;
+        std::vector<NodeID> node_ids_;
+        std::vector<double> timestamps_;
```

Comment:
As we are now keeping the raw arrays in memory I think we can drop the `spikes_` struct and build it on the fly when sorting/filtering is needed